app/assets/stylesheets/v3/_settings.scss

```diff
@@ -211,8 +211,10 @@ $accordion-content-padding: 1rem;
 // 8. Accordion Menu
 // -----------------
 
-$accordionmenu-arrows: false;
-$accordionmenu-arrow-color: $primary-color;
+
+$accordionmenu-arrows: true;
```

Comment:
so i've noticed this keeps getting changed back and forth between our pr's. I had set it to true, then dave set it to false, now you're putting it back to true. So I think maybe check with Dave on why he needed this to be false, I thought I asked him about it in his PR but I'm not sure it was addressed.

Reply:
I'm okay with it being true, I ended up overriding it manually using the classes.

Reply:
Im okay with it being true, too